editors/vscode/src/common.ts

```diff
@@ -130,6 +123,18 @@ export function languageClientOptions(
     };
 }
 
+export function use_wasm_preview(): boolean {
+    if (process.env.hasOwnProperty("CODESPACES")) {
+        return true;
```

Comment:
This does something different than the code it replaces.  
The original code was changing the default, while still allowing the user to override.
(My assumption was that the "CODESPACES" heuristic might not have been perfect)
Now it forces the wasm preview when this env variable is set.  
I guess that's fine too as it is unlikely that the user want to change that if using codespaces



The most problematic difference is that this function seems to only be called when using the "Show Preview" action from the command palette.  But this is not what is called when using the lense, if i'm not mistaken. 
But then the default would still to show the native preview in codespaces when clicking on the lense or the code action. that doesn't work.